### PR TITLE
BRO Titania, Voice of Gaea & Titania, Voice of Gaea

### PR DESCRIPTION
**Pull request by Simisays**
Thanks to The Marvel From Mercury from Discord !

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/argoth_sanctum_of_nature.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Argoth, Sanctum of Nature
+ManaCost:no cost
+Types:Land
+K:ETBReplacement:Other:LandTapped
+SVar:LandTapped:DB$ Tap | Defined$ Self | ETB$ True | ConditionPresent$ Creature.Legendary+YouCtrl | ConditionCompare$ EQ0 | SpellDescription$ CARDNAME enters the battlefield tapped unless you control a legendary green creature
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/argoth_sanctum_of_nature.txt:5]: Missing . (in Oracle too)

File: forge-gui/res/cardsfolder/upcoming/titania_voice_of_gaea_titania_gaea_incarnate.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
+Name:Titania, Voice of Gaea
+ManaCost:1 G G
+Types:Legendary Creature Elemental
+PT:3/4
+K:Reach
+T:Mode$ ChangesZoneAll | ValidCards$ Land.YouOwn+nonToken | Origin$ Any | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigLifegain | TriggerDescription$ Whenever one or more land cards are put into your graveyard from anywhere, you gain 2 life.
+SVar:TrigLifegain:DB$ GainLife | Cost$ G | LifeAmount$ 2
+T:Mode$ Phase | Phase$ Upkeep | CheckSVar$ X | SVarCompare$ GE4 | IsPresent$ Card.Self+YouOwn | IsPresent2$ Land.YouCtrl+YouOwn+namedArgoth; Sanctum of Nature | ValidPlayer$ You | Execute$ Meld | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, if you both own and control CARDNAME and a land named Argoth, Sanctum of Nature, exile them, then meld them into Titania, Gaea Incarnate.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/titania_voice_of_gaea_titania_gaea_incarnate.txt:8]: TriggerDescription incomplete

File: forge-gui/res/cardsfolder/upcoming/argoth_sanctum_of_nature.txt (hunk 1 above)
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/argoth_sanctum_of_nature.txt:5]: Green missing

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/argoth_sanctum_of_nature.txt]: No still missing: look at condition

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/argoth_sanctum_of_nature.txt]: oh thought about the card itself, fixed

**Review by tool4ever — APPROVED** [review]
thanks!

**Pull request merged**